packages/itwinui-css/src/select/select.scss

```diff
@@ -17,23 +17,32 @@
   min-block-size: var(--_iui-select-min-height);
   padding-block: var(--_iui-select-padding-block);
   padding-inline: var(--iui-size-s);
-  gap: var(--iui-size-xs);
   transition: border-color var(--iui-duration-1) ease-out;
-  display: flex;
-  align-items: center;
   user-select: none;
-  position: relative;
-  overflow: hidden;
   color: var(--_iui-input-text-color);
   background-color: var(--_iui-input-background-color);
   border: 1px solid var(--_iui-input-border-color);
   line-height: var(--iui-size-l);
+  font-size: var(--iui-font-size-1);
+  font-family: var(--iui-font-sans);
```

Comment:
These could be combined as `font: inherit`

Reply:
Done!